src/main/java/com/workoutbuilder/enterprise/service/IExerciseService.java

```diff
@@ -51,5 +51,11 @@ public interface IExerciseService {
      */
     List<Exercise> findExercisesByWorkoutId(long workoutId);
 
+    /**
+     * Find exercises by a workout name.
+     *
+     * @param name The name of the workout to be found.
+     * @return The exercise with the given ID.
+     */
```

Comment:
I added a Javadoc to this method. I was hoping it provided some useful information on the on this method.
Source: https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3